### DISCUSSION TITLE
removing 'mail.' from the install domain name

### DISCUSSION THREAD

**Discussion by shawty** [modoboa/modoboa#3157] [Installer]
Is there a way to remove the "mail." bit from the system domain name when installing.

The host name for my server has already been allocated, so when I enter EG: "mailsys.mydomain.net" it needs to stay like that, and NOT be changed to "mail.mailsys.mydomain.net"

I understand adding the prefix if only the TLD portion of the domain is given, but if I supply the host name too then it should accept what I've entered verbatim, is there a way to make the installer do things this way?

**Comment by tonioo** [comment]
Use the --stop-after-configfile-check option the first time your run the installer and update the configuration file (installer.cfg)

**Reply by shawty** [reply]
thanks @tonioo I'll try that

Basically I need for the FDQN of the system once installed, to be installed as   "mailsys.mydomain.tld", as I'm installing this alongside an already installed system called "mail.mydomain.tld"

Cheers
Shawty
